tvos-types.d.ts

```diff
@@ -111,4 +111,8 @@ declare module 'react-native' {
   }
 
   export class TVTextScrollView extends React.Component<TVTextScrollViewProps> {}
+  
+  export interface PressableStateCallbackType {
+    readonly focused: boolean;
```

Comment:
Does this overwrite the Typescript type for the `pressed` state, which presumably already exists in React Native core types?

Reply:
nope! they are merged, not overwritten: https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces the result is that PressableStateCallbackType has both focused and pressed properties.